tests/dns/dns-rcode/test.rules

```diff
@@ -0,0 +1,5 @@
+# Should alert in client direction.
+alert dns any any -> any any (dns.rcode:3; flow:to_client; sid:1; rev:1;)
```

Comment:
Since `rcode` can match on both directions, would be good if at least one of the rules showed that, or at least was for `flow:to_server` :)